spicedb-binding/src/main/java/com/oviva/spicegen/spicedbbinding/internal/CheckPermissionMapper.java

```diff
@@ -29,4 +35,27 @@ public CheckPermissionRequest map(CheckPermission checkPermission) {
         .setPermission(checkPermission.permission())
         .build();
   }
+
+
+  public CheckBulkPermissionsRequest mapBulk(List<CheckPermission> checkPermissions) {
+    // make sure all consistency tokens are the same, or default to fully consistent
```

Comment:
Let me know what you think about this, 
The added benefit is that this can reuse the checkPermission methods generated on the models